addons/mail/static/src/views/web/fields/many2many_tags_email/many2many_tags_email.js

```diff
@@ -149,13 +102,31 @@ export class FieldMany2ManyTagsEmail extends Many2ManyTagsField {
      * @param {Record} record
      */
     onTagClick(event, record) {
-        if (this.props.canEditTags) {
-            return this.openMany2xRecord({
-                resId: record.resId,
-                context: this.props.context,
-                title: _t("Edit: %s", record.data.name),
-            });
-        }
+        const viewProfileBtnOverride = () => {
+            const action = {
+                type: "ir.actions.act_window",
+                res_model: "res.partner",
+                res_id: record.resId,
+                views: [[false, "form"]],
+                target: "current",
+            };
+            this.actionService.doAction(action);
+        };
+        this.recipientsPopover.open(event.target, {
+            viewProfileBtnOverride,
+            id: record.resId,
+        });
+    }
+
+    async quickCreateRecipient(request, params) {
+        const [name, email] = parseEmail(request);
+        const [partnerId] = await this.orm.create("res.partner", [{ name, email }]);
+        return this.props.record.data[this.props.name].addAndRemove({ add: [partnerId] });
+    }
+
+    async updateRecipient(newEmail, partnerId) {
+        await this.orm.write("res.partner", [partnerId], { email: newEmail });
+        await this.props.record.save();
```

Comment:
remove this line (which is what causes the tag to disappear in the Calendar delete wizard).
Why was it there in the first place? Do we need to save the fake record in the wizard?